### PR TITLE
Add Client skeleton for endpoints

### PR DESCRIPTION
**Pull request by convoliution**
```python
client = Client()
...
run = client.set_experiment_run()
run.log_model(...)
run.log_requirements(...)
 
from verta.deployment import DirectUpdateStrategy
endpoint = client.get_or_create_endpoint("/production-prediction")
endpoint.update(run, DirectUpdateStrategy())

endpoint.get_status()
```

### CODE REVIEW

File: client/verta/verta/client.py [hunk 1]
```diff
@@ -873,3 +874,32 @@ def registered_models(self):
...
+        workspace = self._set_from_config_if_none(workspace, "workspace")
+        if workspace is None:
+            workspace = self._get_personal_workspace()
+
+        raise NotImplementedError
+
+    def set_endpoint(self, *args, **kwargs):
+        return self.get_or_create_endpoint(*args, **kwargs)
+
+    def endpoints(self, workspace=None):
```
Comment (conradoverta) [inline client/verta/verta/client.py]: Property? Consistency.

File: client/verta/verta/deployment.py [hunk 2]
```diff
@@ -14,6 +14,21 @@
 from ._internal_utils import _utils
 
 
+# TODO: move these to verta/deployment/strategies if restructuring
```
Comment (conradoverta) [inline client/verta/verta/deployment.py]: Why don't we do this right now? Seems like a small effort now vs larger effort later, no?

Comment (convoliution) [reply client/verta/verta/deployment.py]: Sure! I can spend some time doing this.

File: client/verta/verta/client.py (hunk 1 above)
Comment (convoliution) [reply client/verta/verta/client.py]: Whoops, negligence on my part

**Review by conradoverta — APPROVED** [review]

**Pull request merged**